src/Microsoft.Extensions.DiagnosticAdapter/project.json

```diff
@@ -1,6 +1,15 @@
 {
   "version": "1.0.0-*",
-  "description": "A DiagnosticListener implementation which provides duck-typing through generated proxy types.",
+  "description": "Microsoft extension adapter feature to extend DiagnosticListener. Contains DiagnosticListener extension methods and types to enable duck-typing based event handling through generated proxy types.\r\nCommonly used DiagnosticListener extension method:\r\nSystem.Diagnostics.SubscribeWithAdapter",
+  "packOptions": {
+    "tags": [
+      "diagnosticadapter",
+      "diagnosticlistener",
+      "diagnostics",
+      "microsoftextensions",
+      "microsoft"
```

Comment:
Don't need these two `Microsoft*` tags - they don't add anything meaningful because it's just the owner/namespace.